appinfo/routes.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @copyright Copyright (c) 2021 Frederic Ruget (douzebis) <fred@atlant.is>
+ *
+ * @author Frederic Ruget (douzebis) <fred@atlant.is>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+return [
+	'ocs' => [
+		[
```

Comment:
Need to be documented in https://docs.nextcloud.com/server/latest/developer_manual/client_apis/OCS/index.html

Maybe you don't need to create an OCS API with a stable API, if you only want the webui to have access to the files and not also the other clients.